Cargo.toml

```diff
@@ -27,7 +30,9 @@ clap = { version = "4.3", features = ["derive"] }
 criterion = "0.5.1"
 csv = "1.2.2"
 dbase = "0.4"
-diesel = { version = "2.1.0", default-features = false, features = ["postgres"] }
+diesel = { version = "2.1.0", default-features = false, features = [
```

Comment:
@sergiomeneses please disable or adjust your toml auto-formatter - while i usually like those, the newlines it introduces here and above are not very common in rust -- especially in the dependencies list.

Reply:
Done.